### DISCUSSION TITLE
Suddenly I get notifications displayed

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#624] [Q&A] [answered]
Since yesterday, I have suddenly been getting notifications even though I have suppressed them. Even restarting the smartphone does not solve the problem. I'm using Cruiser 5.0.0-b10.
![2024-08-27 20 28 06](https://github.com/user-attachments/assets/07d54e2c-252f-412f-a357-eed8c847394a)
![2024-08-27 20 29 09](https://github.com/user-attachments/assets/bc0cb588-7747-410c-b288-769eb1cc1d3d)
![2024-08-27 20 29 58](https://github.com/user-attachments/assets/0426cc51-2c7d-45e9-b36d-ba0e255ec4c6)

Is there a condition other than the specification in Settings/Navigation/Advanced that activates the notifications?

**Comment by Ciutnik** [comment]
@WolfgangZ42 
I think it's a new Android feature (artificial intelligence perhaps?). It converts speech into text! I also noticed something like this (with another "talking" app), after updating my Samsung phone.

**Comment by devemux86** [comment]
@WolfgangZ42 

These do not seem like the Cruiser navigation notifications.
Please see the image below for how they appear (they have specific icon).

Perhaps they are from another app?

<img src="https://github.com/user-attachments/assets/61af66ce-9510-452d-a592-6b397844f7da" width="50%"/><img src="https://github.com/user-attachments/assets/8468d55c-35e9-46e0-870a-dbc544b6a07e" width="50%"/>

**Comment by Ciutnik** [comment]
I confirm what I wrote earlier. Find something in the settings that resembles "Live subtitles". It used to look completely different, but an update changed that.... and turned it on without asking.

**Reply by devemux86** [reply]
On original Android 14 there is something similar in "Accessibility" settings named "Live Caption".

**Reply by Ciutnik** [reply]
There has been a similar feature for a very long time. I think the new thing is the ability to translate on the fly into your language.

**Reply by WolfgangZ42** [reply, marked as answer]
Sorry for reporting a nonexisting error. It was indeed an accidentally switched on function on my new Pixel 8. Thanks for your tipps!

**Reply by devemux86** [reply]
@WolfgangZ42 

Pixel 8, what do you think about it?

**Reply by WolfgangZ42** [reply]
Not being the smartphone guru, I selected my new phone (the old one smashed when it fell - no, not while riding a motorbike!) depending on the criteria: waterproof, bright display and long-term support for new Android releases. So few weeks ago I got a Pixel 8, after years with various Sony Xperia versions. What should I say? It does its job. I don't invest time to learn all the gimmics (A10 to A14 is a reasonable step). Brightness is not significantly better than at the Sony XZ2. The photos are very good - but I still use a "real" camera to take pictures. So I'm satisfied without having high expectations.

**Comment by devemux86** [comment]
Latest Android or Google updates also display an annoying menu icon when we write text:

<img src="https://github.com/user-attachments/assets/9eac4cab-0562-4426-9695-d4d030e9dfa6" width="50%"/><img src="https://github.com/user-attachments/assets/5bc5d51a-fece-4111-a0a8-8efe083959fb" width="50%"/>

**Reply by Ciutnik** [reply]
This is actually annoying. Does this icon appear when an external keyboard is connected? Android on-screen keyboards have their own menus, but they don't seem to have an "Alt" key 🙂

**Reply by devemux86** [reply]
Indeed there was an external keyboard. :+1:
